views/assets/js/govuk_frontend_toolkit/stageprompt.js

```diff
@@ -0,0 +1,69 @@
+// Stageprompt 2.0.1
```

Comment:
INFO - My guess is all this govuk stuff is library. If it is a library, why are we not using it through npm but hosting and serving ourselves?